GoogleDataTransport/GDTCORLibrary/Public/GDTCORSqlite.h

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2018 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+#import <sqlite3.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/** Convenience typedef to define a block ran on each row of a query result. */
+typedef void (^GDTCORSqliteRowResultBlock)(void);
+
+/** Instantiates a sqlite3 object given the filename.
+ *
+ * @param db The db reference to instantiate.
+ * @param fileURL The file URL to make a db at.
+ * @return YES if the db was successfully opened, NO otherwise.
+ */
+BOOL GDTCORSQLOpenDB(sqlite3 *_Nullable *_Nullable db, NSURL *fileURL);
```

Comment:
I don't see any of these methods are used. Is this just adding the helper functions so far?

Reply:
Yeah, these are just helper functions. They're only used in tests right now.